### DISCUSSION TITLE
Generating Response with MiniCPM-V

### DISCUSSION THREAD

**Discussion by forpr1093** [oobabooga/text-generation-webui#5560] [Q&A]
I have loaded this model: https://huggingface.co/openbmb/MiniCPM-V with transformers (trust0remote0code activated) into webui.
However, when I try to generating response, it gave me this error:
![image](https://github.com/oobabooga/text-generation-webui/assets/101260638/9f0bb3a1-7003-42c7-84d2-d4c05f137794)

Any help will be appreciated!

**Comment by thistleknot** [comment]
minicpm has been incorporated into the latest version of llama.cpp (yet I'm not sure exactly how to use it with the non openbmb llama.cpp).  Any thoughts on supporting the model for use with text-generation-webui?
